gprofiler/utils/__init__.py

```diff
@@ -378,6 +403,10 @@ def grab_gprofiler_mutex() -> bool:
     on filesystem structure (as happens with file-based locks).
     In order to see who's holding the lock now, you can run "sudo netstat -xp | grep gprofiler".
     """
+    # Bypass due to lack of Windows support
+    return True
+    # End Edit
```

Comment:
Under `if is_windows()` and add a TODO for us to fix later (we'll implement the mutex in some other way)

Reply:
Done